about/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: About the LLNL Software Portal
+layout: default
+---
+
+## {{ page.title }}
+{: .page-header}
+
+Welcome to the Lawrence Livermore National Laboratory software portal! The purpose of this software portal (or catalog if you prefer) is to serve as the definitive source of truth pointing to all software that has been produced and released by Lawrence Livermore National Laboratory.
```

Comment:
"definitive source of truth" and "all software that has been produced and released" are inaccurate.  I think you should tone this part down.  The portal is for open source software (we release stuff that's not open source too), and "definitive source of truth" sounds overly strong.  Why not "This software portal is LLNL's official hub for open source software" or something similar?